chat-plugins/faq.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const ROOMFAQ_FILE = path.resolve(__dirname, '../config/chat-plugins/faqs.json');
+
+let roomFaqs = {};
+try {
+	roomFaqs = require(ROOMFAQ_FILE);
+} catch (e) {} // file doesn't exist or contains invalid JSON
```

Comment:
Odd, there didn't seem to be a problem with it when Zarel helped me write it https://github.com/Zarel/Pokemon-Showdown/pull/1239#issuecomment-57916279
Regardless, this code for loading the json doean't crash, but it doesn't handle situations like the user who launched PS lacking the right privileges to edit the file, or the file already being used by another process. If you want to do it the best way, by all means fix it rather than arguing over precedent.